src/main/java/com/base64/d/StudyGuideChecker.java

```diff
@@ -8,127 +8,120 @@
 
 import java.io.IOException;
 import java.util.Base64;
-import java.util.Date;
 import java.util.HashSet;
 import java.util.Set;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 public class StudyGuideChecker {
-    int count = 0;
-    public int totalLinksFound = 0;
-    String url3 = "https://content.infinitecampus.com/sis/latest/study-guide/ad-hoc-filters-letters-and-data-viewer";
-    String decodedText = "";
-    String iframeSrc = "";
-    String date = new Date().toString();
 
-    String titleOfStudyGuide = "";
-    Set<String> studyGuideLinks = new HashSet<>();
-    Set<String> badLinks = new HashSet<>();
+    public void checkStudyGuides() {
+        int count = 0;
+        int totalLinksFound = 0;
 
-    WriteToFile writeToFile = new WriteToFile();
+        String decodedText = "";
+        String iframeSrc = "";
+        String titleOfStudyGuide = null;
 
-    public void checkStudyGuides() {
+        Set<String> studyGuideLinks = new HashSet<>();
+        Set<String> badLinks = new HashSet<>();
 
-        for (String link2 : AllStudyGuideLinks.ALL_STUDY_GUIDES) {
+        WriteToFile writeToFile = new WriteToFile();
+
+        for (String studyGuideLink : AllStudyGuideLinks.ALL_STUDY_GUIDES) {
+            // Find all links within a Study Guide
             try {
-//
-//          Connect to the study guide url
-                Document doc3 = Jsoup.connect(link2).get();
+                // Connect to the study guide url
+                Document wraperDocument = Jsoup.connect(studyGuideLink).get();
 
-//            grab the iframe
-                Element iframe = doc3.select("iframe").first();
```

Comment:
A lot of the variables have more descriptive names, which is helpful. I've renamed `doc3` and `link2` to be more descriptive of what they will actually hold. Good naming is a best practice in general, and is really helpful when others read your code.